src/UE/Application/States/ReceivingCallState.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "BaseState.hpp"
+namespace ue
+{
+
+class ReceivingCallState : public BaseState
+{
+public:
+    ReceivingCallState(Context &context, const common::PhoneNumber& peer);
+    void handleTimeout() final;
+    void handleCallRequest(common::MessageId msgId,
+                           const common::PhoneNumber& peer,
+                           const std::string &enc) final;
+    void handleCallDrop() final;
+    void handleCallAccept() final;
+    void handleCallDropped() final;
```

Comment:
I would replace `final` with `override`, since it matches the rest of the project.